README.md

```diff
@@ -1,3 +1,9 @@
+[![forthebadge](https://forthebadge.com/images/badges/made-with-python.svg)](https://forthebadge.com)
+[![forthebadge](https://forthebadge.com/images/badges/it-works-why.svg)](https://forthebadge.com)
+[![forthebadge](https://forthebadge.com/images/badges/not-a-bug-a-feature.svg)](https://forthebadge.com)
```

Comment:
I'd say not to add this one as its in red which makes it look like things are failing in the repo; `not-a-bug-a-feature.svg`